app/main.py

```diff
@@ -1,8 +1,21 @@
+
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        name = person["name"]
+        if "wife" in person and person["wife"]:
+            Person.people[name].wife = Person.people[person["wife"]]
+        elif "husband" in person and person["husband"]:
+            Person.people[name].husband = Person.people[person["husband"]]
```

Comment:
The code assumes that the spouse (wife or husband) is already present in the `Person.people` dictionary. If the spouse is not found, this will raise a `KeyError`. Consider adding a check to ensure the spouse exists in the dictionary before assigning it, or handle the exception gracefully.